aries-site/src/examples/templates/page-layouts/anatomy/PageMarginAnatomy.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Box, Grid, Stack, Text } from 'grommet';
+import PropTypes from 'prop-types';
+
+// values chosen to best visually represent the margin area in diagram
+const largeColumns = ['13px', 'auto', '13px'];
+const smallColumns = ['8px', 'auto', '8px'];
+
+const MarginText = ({ label }) => (
+  <Stack anchor="bottom">
+    <Text alignSelf="center">{label}</Text>
+    {/* values chosen to best visually represent horizontal line in diagram. */}
+    <Box width="12px" height="1px" background="black" />
+  </Stack>
+);
+
+const GridViewPort = ({ columns, label, marginLabel, width }) => (
+  <Box gap="small" width={width}>
+    <Box direction="row" justify="between">
+      <MarginText label={marginLabel} />
+      <MarginText label={marginLabel} />
+    </Box>
+    <Grid
+      columns={columns}
+      // values chosen to best visually represent the margin area in diagram
+      rows={['20px', 'small', '20px']}
+      areas={[
+        ['left-margin', 'top-gap', 'right-margin'],
+        ['left-margin', 'viewpoint', 'right-margin'],
```

Comment:
```suggestion
        ['left-margin', 'viewport', 'right-margin'],
```
should this be "viewport" instead?

Reply:
aw yes thank you